File: Android/EmotionsApp/EmotionsApp/Models/DTOs/Content/Album/AlbumResponseDTO.swift
//
//  AlbumResponseDTO.swift
//  EmotionsApp
//
//  Created by Муса Зарифянов on 09.11.2025.
//

import Foundation

struct AlbumResponseDTO: Codable, Sendable {
    var id: Int64
    var title: String
    var description: String
    var createdAt: Date
    var topics: [TopicShortDTO]
}

File: emotion-back/content/src/main/java/com/example/content/dtos/album/AlbumResponseDto.java
package com.example.content.dtos.album;

import com.example.content.dtos.topic.TopicShortDto;
import lombok.Builder;
import lombok.Data;

import java.util.Date;
import java.util.Set;

@Data
@Builder
public class AlbumResponseDto {

    Long id;
    String title;
    String description;
    Long userId;
    Date createdAt;
    Set<TopicShortDto> topics;
}
